students/jungmin/westagram/users/views.py

```diff
@@ -0,0 +1,42 @@
+import json
+import re
+
+from django.db import IntegrityError
+from django.http import JsonResponse
+from django.views import View
```

Comment:
```suggestion
from django.db    import IntegrityError
from django.http  import JsonResponse
from django.views import View
```
조금 더 가독성 좋게 할 수 있겠습니다.